packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -90,22 +111,33 @@ export class RemoteChannelContext implements IChannelContext {
         this.services.deltaConnection.setConnectionState(connected);
     }
 
+    public rebaseOp(message: ISequencedDocumentMessage, localOpMetadata: unknown) {
+        if (this.isLoaded) {
+            this.services.deltaConnection.rebase(message, localOpMetadata);
```

Comment:
I think it's better to use same naming, i.e. rebaseOp here and on deltaConnection.